modules/afsocket/transport-mapper-inet.c

```diff
@@ -107,19 +108,118 @@ transport_mapper_inet_init(TransportMapper *s)
 {
   TransportMapperInet *self = (TransportMapperInet *) s;
 
-  if (self->tls_context && !tls_context_setup_context(self->tls_context))
+  if (self->tls_context && (tls_context_setup_context(self->tls_context) != TLS_CONTEXT_SETUP_OK))
     return FALSE;
 
   return TRUE;
 }
 
+typedef struct _call_finalize_init_args
+{
+  TransportMapperInet *transport_mapper_inet;
+  TransportMapperAsyncInitCB func;
+  gpointer func_args;
+} call_finalize_init_args;
```

Comment:
Structure names are written in camelcase.

Reply:
This is an internally used, private helper structure, not acting as a "real" class, so this naming is intentional (I know that there is no such naming convention in syslog-ng, but in this case I prefer this style, and there are one or two other structs across the  code that are not following the camelcase style).